x/ibc-transfer/client/cli/tx.go

```diff
@@ -16,9 +16,9 @@ import (
 )
 
 const (
-	flagTimeoutHeight    = "timeout-height"
-	flagTimeoutTimestamp = "timeout-timestamp"
-	flagAbsoluteTimeouts = "absolute-timeouts"
+	flagRelativeTimeoutHeight    = "relative-timeout-height"
```

Comment:
I think `ibc-timeout-height` might actually be better. Or maybe there's a better approach that can be addressed in a different pr.

Currently these flags are used in conjunction with `flagAbsoluteTimeouts`, if that flag is false, they are used as relative, if it is true they are used as absolute timeouts. 

The reason for initially not just making this 4 flags in total is, it becomes a little messy to tell when a user wants to disable flags while still allowing relative timeouts with sane default values to be the default usage. If you introduced absolute value flags, to disable a timeout you'd have to set --relative-timeout-height and --absolute-timeout-height to 0. Maybe this isn't so bad. I guess you could introduce logic like if any absolute timeout flag is used the relatives are dismissed. 

@fedekunze thoughts?